src/main/java/screens/Window.java

```diff
@@ -0,0 +1,143 @@
+package screens;
+
+import filesystem.FileSystem;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Menu;
+import javafx.scene.control.MenuBar;
+import javafx.scene.control.MenuItem;
+import javafx.scene.layout.BorderPane;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+import logging.Logger;
+import logging.LoggerFactory;
+
+import java.io.File;
+
+/**
+ * Main application.
+ */
+@SuppressWarnings("FieldCanBeLocal")
+public class Window extends Application{
```

Comment:
This line contains a checkstyle error ({ not preceded by whitespace) please fixcheckstyle errors before making a PR